cypress/PageObjects/addNewElement.js

```diff
@@ -0,0 +1,30 @@
+///reference types="Cypress"
+
+class addNewElement {
+
+    get CasePage(){
+        return cy.get('[aria-label="Case-Mate Tough Clear Case for Pixel 6a* (result 1 of 2)"]').click()
```

Comment:
selectProduct(productName)
it is not a locator, it is a method with parameter - productName